### PR TITLE
Simple container for Onnxrt + MIGraphX + Torch for Docker…

### PR DESCRIPTION
**Pull request by TedThemistokleous**
… Distro Generation|

Skip CI

Simple just install all the pieces to get a working container for end to end with Onnxruntime  + pytorch + MIGraphX under one roof.

Torch is required here as Onnxruntime leverages existing torch functionality for some of the benchmarking scripts and things.

Will pull out of draft once 6.0.2 is available to use and updated here

### CODE REVIEW

**Comment by TedThemistokleous** [comment]
This docker is what will be used and then uploaded to docker hub on release.

**Comment by TedThemistokleous** [comment]
Tested image and everything appears a-ok when running versions of things

image

File: tools/docker/migraphx_with_onnxruntime_pytorch.docker [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+RUN amdgpu-install --usecase=rocm -y && rm *.deb
+
+#Install MIGraphX from package manager
+RUN apt install -y migraphx
+
+#Pieces for Onnxruntime for ROCm and MIGraphX Execution Provider Support
+RUN wget https://repo.radeon.com/rocm/manylinux/rocm-rel-6.0.2/onnxruntime_rocm-inference-1.17.0-cp310-cp310-linux_x86_64.whl
+RUN pip3 install *.whl && rm -rf *.whl
+
+#Pieces for Pytoprch
```
Comment (umangyadav) [inline tools/docker/migraphx_with_onnxruntime_pytorch.docker:20]:
```suggestion
#Pieces for pytorch
```

**Review by umangyadav — APPROVED** [review]
LGTM. except typo

**Review by shivadbhavsar — APPROVED** [review]

**Review by causten — APPROVED** [review]

**Pull request merged**